### ISSUE TITLE
Support task dependencies

### ISSUE THREAD

**Issue by fniessink** [fniessink/next-action#101]
Dependencies are not part of the core todo.txt format. What do other tools use? Some id:/after: mechanism probably.

Check out:

- [x] https://github.com/todotxt/todo.txt/issues/8: doesn't really mention what common dependency tags are
- [x] https://github.com/bram85/todo.txt-tools/wiki/Dep-%28action%29: repo archived, id/p(arent), task with p:<id> is dependent on/must be done before task with id:<id>
- [x] http://todotxt.yahoogroups.narkive.com/LS01SVbB/tasks-dependency-in-todo-txt: design notes only
- [x] https://www.topydo.org/static.html#Dependencies: id/p(arent), same as item 2.
